README.md

```diff
@@ -9,47 +9,66 @@
 
-Location logs provide time-stamped records of where users have been, allowing them to share information with health officials accurately and quickly. This helps support contact tracing efforts to slow the spread of the virus.
+## Privacy Preserving 
 
-What’s truly special about Safe Paths, though, is its privacy protection.
+What’s truly special about Safe Paths is our strong commitment to preserving the privacy of individual users. We're building an application that can help contain outbreaks of COVID-19 without forcing users to sacrafice their personal privacy. For example, if a user chooses to use Safe Paths to maintain a record of their locations use a time stamped GPS log, which stores 14 days of data in 5 minute increments. The location log is stored on the users phone. The logged data only leave the device if the user chooses to send the information to an authorized public health authority (PHA) as part of the contact tracing prcoess.
 
-Data never leaves a user's device without their password entry and explicit consent. The location log generated by Safe Paths cannot be accessed from outside the user’s device, meaning data transfer occurs only if the user chooses to share it with a researcher or health official.
+## Multiple Capabilities
 
-<img align="right" src="./assets/PreviewUI.png" data-canonical-src="./assets/PreviewUI.png"/>
+Safe Paths is designed to support a range of DCT and public health use cases. Currently the main build uses GPS for location tracking. Our roadmap includes adding support for other location and proximity technologis, symptom tracking, and communication with PHAs. 
 
-## Overview
+### Project Aurora and the Google Apple Exposure Notification API (GAEN) 
 
-Safe Paths is a ‘privacy-first’ app that allows you to log your GPS trails on your own phone. The information is stored locally and never shared with anyone (not even with us or MIT) until you explicitly decide to manually export the data.
+Project Aurora is the code name for a build from Safe Paths for an app that is fully compliant with GAEN. This build will not include GPS, and it will be compliant with all the rules and requirements for GAEN apps. [Learn more about Project Aurora.](https://covidsafepaths.org/a-new-open-source-solution-for-the-google-apple-exposure-notification-api/) 
 
-The location log generated by Safe Paths cannot be accessed from outside the user’s device. However, the user can import and export their location information and use it in other projects and applications.
+## Multiple Implementation Strategies 
 
-Safe Paths logs your device’s location once every five minutes and stores 14 days of data.
+Safe Paths is designed to support a variety of implementation strategies. 
```

Comment:
The Safe Paths app is being developed to support a variety of build 'flavors' of the application around core health and tracing functionality. Reach out to our team to discuss creating a `flavor` for your use-case.